appveyor.yml

```diff
@@ -1,71 +1,84 @@
 version: '{build}'
 
-os: Visual Studio 2015
-
 environment:
   matrix:
-  - Toolset: v140
-  - Toolset: v120
-  - Toolset: v110
-  - Toolset: v100
+    - compiler: msvc-10-seh
+      generator: "Visual Studio 10 2010"
+
+    - compiler: msvc-11-seh
+      generator: "Visual Studio 11 2012"
+
+    - compiler: msvc-11-seh
+      generator: "Visual Studio 11 2012 Win64"
+
+    - compiler: msvc-12-seh
+      generator: "Visual Studio 12 2013"
+
+    - compiler: msvc-12-seh
+      generator: "Visual Studio 12 2013 Win64"
 
-platform:
-  - Win32
-  - x64
+    - compiler: msvc-14-seh
+      generator: "Visual Studio 14 2015"
+
+    - compiler: msvc-14-seh
+      generator: "Visual Studio 14 2015 Win64"
+
+    - compiler: gcc-5.3.0-posix
+      generator: "MinGW Makefiles"
 
 configuration:
-#  - Release
   - Debug
+  - Release
```

Comment:
I disabled this configuration when created appveyor.yml. Now I think it's better to check both configurations